### PR TITLE
ceph.spec.in: package prometheus default alerts for SUSE

### PR DESCRIPTION
**Pull request by jan--f**
Signed-off-by: Jan Fajerski <jfajerski@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
@b-ranto We're aiming to get this into nautilus soonish, hence the initially SUSE-only approach here, but we're open to collaboration as always ;-) 

The question, I guess, is whether it makes sense to ship this file on distros other than SUSE. And, if so, where should it go on those distros?

File: ceph.spec.in [hunk 1]
```diff
@@ -1058,6 +1060,15 @@ collecting data from Ceph Manager "prometheus" module and Prometheus
 project "node_exporter" module. The dashboards are designed to be
 integrated with the Ceph Manager Dashboard web UI.
 
+%if 0%{?suse_version}
+%package prometheus-alerts
+Summary:        Prometheus alerts for a Ceph deplyoment
+BuildArch:      noarch
+Group:          System/Monitoring
+%description
```
Comment (smithfarm) [inline ceph.spec.in]: This line needs to be `%description prometheus-alerts` for the build to succeed. Without it, we get:

```
[   80s] warning: line 1070: second Description
[   81s] error: Package has no %description: ceph-prometheus-alerts
```

**Comment by smithfarm** [comment]
**test build in progress** https://shaman.ceph.com/builds/ceph/package-prometheus-alerts-suse/df22cd03b8511b6aee87b1eada73e0cbd2c30944 (master plus the commit in this PR)

**Comment by smithfarm** [comment]
@jan--f Note that Shaman only builds ceph-ci branches that start with the string `wip-` so I had to prepend that string when I pushed your branch to ceph-ci... might be easier to just start all your wip branches with it?

**Comment by jan--f** [comment]
@smithfarm noted

**Comment by smithfarm** [comment]
https://shaman.ceph.com/builds/ceph/wip-package-prometheus-alerts-suse/317232e6ce999679ba710d18b4cabccce0f4e1ec

**Pull request merged**